docs/accessibility.md

```diff
@@ -0,0 +1,9 @@
+# Accessibility
+
+The **Microsoft MakeCode** team wants to be sure that every person on the planet can play with it to achieve more. It is very important for us when we're working to make computer science more accessible to consumers with disabilities.
+
+Here are the key accessibility features of **Microsoft MakeCode**:
+
+* [Keyboard Navigation](/accessibility/keyboard-navigation)
+* [Assistive technologies](/accessibility/screen-readers)
+* [High Contrast](/accessibility/high-contrast)
```

Comment:
Add link to developer section

Reply:
I'm not convinced if it's a good idea. I don't like the idea to add a link to the developer section in a part reserved to end users. Imagine a kid who find it because it's in the end users documentation, start to read it, and of course understand nothing. He might feel lost and abandon.